src/consts/permissions.ts

```diff
@@ -114,40 +114,40 @@ enum PERMISSIONS {
   ScormCreate = 'scorm_create',
   ScormUpdate = 'scorm_update',
   ScormDelete = 'scorm_delete',
-  ScormGetTrack = 'scorm_get_track',
-  ScormSetTrack = 'scorm_set_track',
+  ScormGetTrack = 'scorm_track-read',
+  ScormSetTrack = 'scorm_track-update',
 
   /* -------- EscolaLMS/Reports -------- */
 
-  ReportList = 'reports_list',
+  ReportList = 'report_list',
 
   /* -------- EscolaLMS/Courses-Import-Export -------- */
 
-  CourseExport = 'courses-import-export_export', // TODO: unused in api
-  CourseImport = 'courses-import-export_import', // TODO: unused in api
+  CourseExport = 'courses-import-export_export',
+  CourseImport = 'courses-import-export_import',
   CourseExportAuthored = 'courses-import-export_export_authored',
 
   /* -------- EscolaLMS/Permissions -------- */
 
-  PermissionRoleManage = 'permissions_role_manage',
-  PermisionRoleList = 'permissions_role_list',
-  PermissionRoleCreate = 'permissions_role_create',
-  PermissionRoleDelete = 'permissions_role_delete',
-  PermissionRoleRead = 'permissions_role_read',
-  PermissionRoleUpdate = 'permissions_role_update', // access to edit permissions
+  PermissionRoleManage = 'permission_role_manage',
+  PermisionRoleList = 'permission_role_list',
+  PermissionRoleCreate = 'permission_role_create',
+  PermissionRoleDelete = 'permission_role_delete',
+  PermissionRoleRead = 'permission_role_read',
+  PermissionRoleUpdate = 'permission_role_update', // access to edit permissions
 
   /* -------- EscolaLMS/Templates -------- */
 
-  TemplateList = 'templates_list',
-  TemplateRead = 'templates_read',
-  TemplateCreate = 'templates_create',
-  TemplateDelete = 'templates_delete',
-  TemplateUpdate = 'templates_update',
+  TemplateList = 'template_list',
+  TemplateRead = 'template_read',
+  TemplateCreate = 'template_create',
+  TemplateDelete = 'template_delete',
+  TemplateUpdate = 'template_update',
 
   /* -------- EscolaLMS/Notifications -------- */
 
   NotificationListAll = 'dashboard-app_notification-list_access', // ability to read all notifications (including those not belonging to you)
-  NotificationList = 'dashboard-app_notification-event-list_access',
+  NotificationList = 'dashboard-app_notification-event-list_access_self',
```

Comment:
tu jest `notification-event` a w poprzednim `notification` a obstawiam, że odnoszą się dokładnie to tego samego tylko z innym modyfikatorem (tu self)

Reply:
ok